doc: Add migration tool troubleshooting section (#1319)
TheThingsIndustries/lorawan-stack-docs: doc/content/the-things-stack/migrating/migration-tool/troubleshooting/_index.md

---
title: "Troubleshooting"
description: "Troubleshooting the Migration tool"
weight:
---

<!--more-->

This section provides help for common issues you may have while using the migration tool.

## error:pkg/redis:not_found (entity not found)

> This error could occur when the device is not properly registered in all {{% tts %}} components(`Identity Server`, `Join Server`, `Network Server`, and `Application Server`) and trying to export the device using the migration tool. Below is the example error:

Reviewer: ```suggestion
If the device is not properly registered in all {{% tts %}} components (Identity Server, Join Server, Network Server and/or Application Server) while trying to export the device using the migration tool, the following error might occur:
```

```

> Error: error:go.thethings.network/lorawan-stack-migrate/pkg/export:export (export device `newdev1`)
> error:go.thethings.network/lorawan-stack-migrate/pkg/export:export (export device `newdev1`)
>     device_id=newdev1
>     correlation_id=e99d3b6b49ce47b08ebbe8f44d57a783
> --- error:pkg/redis:not_found (entity not found)
>     correlation_id=33fabc577f084b6c998ab3b0c1382745

Reviewer: ```suggestion
error:go.thethings.network/lorawan-stack-migrate/pkg/export:export (export device `newdev1`)
error:pkg/redis:not_found (entity not found)
```

Reviewer: I know my suggestion show the full error, but the rest of lines don't really contribute to understanding in any way and influence readability, so there's no much need for them to be there

```

> We suggest you delete such devices from {{% tts %}} Application and try exporting other devices.

Reviewer: If I want to export dev A, how is deleting that device and trying to export device B helping me?

## transport: error while dialing: dial tcp 52.212.223.226:8884: i/o timeout

> The error `transport: error while dialing: dial tcp 52.212.223.226:8884: i/o timeout` indicates that there might be an issue with your network and restricts to establishment of the connection. The issue could be due to your **organization's network firewall blocking the connections** on port `8884`.

Reviewer: ```suggestion
This error indicates that there might be an issue with your network such as your organization's network firewall blocking the connections on port `8884`.
```

Reviewer: Which TTS deployment are we talking about here?

Author: The Things Stack Sandbox and Cloud.
